ambry-api/src/main/java/com/github/ambry/config/RouterConfig.java

```diff
@@ -591,6 +591,11 @@ public class RouterConfig {
   public final long routerBlobMetadataCacheMaxSizeBytes;
   long numBytesInOneMb = (long) Math.pow(1024, 2);
 
+  public static final String ROUTER_SMALLEST_BLOB_FOR_METADATA_CACHE = "router.smallest.blob.for.metadata.cache";
+  @Config(ROUTER_SMALLEST_BLOB_FOR_METADATA_CACHE)
+  public final long routerSmallestBlobForMetadataCache;
+  long numBytesInOneTb = (long) Math.pow(1024, 4);
```

Comment:
we can use a static value for this and change the variable name to all upper case. 

And 1 TB, really?

Reply:
The question is more about why do we differentiate metadata chunk at different size? I don't see why we would not cache some metadata chunk but cache the other.